yb-voyager/cmd/importSchema.go

```diff
@@ -90,23 +90,40 @@ func importSchema() {
 	// Import ALTER TABLE statements from sequence.sql only after importing everything else
 	isAlterStatement := func(objType, stmt string) bool {
 		stmt = strings.ToUpper(strings.TrimSpace(stmt))
-		return objType == "SEQUENCE" && strings.HasPrefix(stmt, "ALTER TABLE")
+		if objType == "SEQUENCE" && strings.HasPrefix(stmt, "ALTER TABLE"){
+			return true
+		} else if objType == "TABLE" && strings.Contains(stmt, "ALTER TABLE") && strings.Contains(stmt, "FOREIGN KEY"){
+			return true
+		} else if objType == "UNIQUE INDEX" && !strings.Contains(stmt, objType){
```

Comment:
are both of these if conditions at line - 97 and 99 required?
Or these can be merged into one?

Reply:
I am not sure if these can be merged, as one is to only run UNIQUE INDEX DDLs for objType UNIQUE INDEX and the other one is to skip only UNIQUE INDEX DDLs at the time of INDEX objType.